app/build.gradle

```diff
@@ -48,9 +48,3 @@ dependencies {
     compile 'com.loopj.android:android-async-http:1.4.5'
     compile files('libs/android-integration-3.1.0.jar')
 }
-
-task appStart(type: Exec, dependsOn: 'installDebug') {
```

Comment:
Well i removed it and app still works fine

On 3 August 2015 at 17:14, Nam Thai <notifications@github.com> wrote:

> In app/build.gradle
> <https://github.com/NamTThai/PokemonShowdownAndroidClient/pull/98#discussion_r36095150>
> :
>
> > @@ -48,9 +48,3 @@ dependencies {
> >      compile 'com.loopj.android:android-async-http:1.4.5'
> >      compile files('libs/android-integration-3.1.0.jar')
> >  }
> > -
> > -task appStart(type: Exec, dependsOn: 'installDebug') {
>
> We don't need this one anymore?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NamTThai/PokemonShowdownAndroidClient/pull/98/files#r36095150>
> .
>



-- 
BOUVET Clément
bouvetcl@gmail.com
 (+33) 6 32 66 03 06